docs/CHANGELOG.asciidoc

```diff
@@ -73,6 +73,13 @@ For information about the {kib} 7.15.0 release, review the following information
 [[known-issue-7.15.0]]
 === Known issue
 
+.Restore *Discover* search sessions with a relative time range
+[%collapsible]
+====
+{kib} is unable to restore *Discover* search sessions with a relative time range. When you restore the *Discover* search session, then run a new search, {kib} displays a `Your search session is still running` message. For more information, refer to {kibana-issue}101430[#101430].
+====
```

Comment:
I think this might work better:

Title: {kib} is unable to restore **Discover** search sessions with a relative time range
Description: When you restore a *Discover* search session, then run a new search, {kib} displays a `Your search session is still running` message. For more information, refer to {kibana-issue}101430[#101430]

Reply:
@jloleysens The content looks good, but please add the formatting.  Put **Discover** in bold (2 places) and use code font for the message, as shown above.

Reply:
Gotcha, that was a copy-paste error :) great catch!